src/mixins/legal-api-mixin.ts

```diff
@@ -52,16 +52,21 @@ export default class LegalApiMixin extends Vue {
       return await this.getDraftFiling()
     } catch (e) {
       // TODO: Throw a flag to the ui from here, if we want to trigger error handling in ui
-      throw new Error('Invalid API Response')
```

Comment:
This line of code would have caught the error thrown by the functions above and re-thrown it, so I agree there's not much benefit here.

What does happen here when an error was thrown -- is it just ignored now?

Another thought is that the throw calls below should be more specific about what call failed (POST, PUT, GET) and/or why they think it's an error; currently they all display the same error :>

Reply:
I'll look into this, i'd like to do something consistent throughout. I'm curious if we have designs/tickets to implement error handling in UI ( dialogs or notifications ). It might be a good thing to do sooner than later!

Reply:
/bcgov/entity#2332

Reply:
😆  Exactly! Don't recall making that..